### PR TITLE
Updates to filtering page

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-180.surge.sh

Updated pages:
- [Filtering](https://neo4j-docs-graphql-180.surge.sh/graphql/5/queries-aggregations/filtering)

File: modules/ROOT/pages/queries-aggregations/filtering.adoc [hunk 1]
```diff
@@ -200,7 +281,9 @@ const features = {
 const neoSchema = new Neo4jGraphQL({ features, typeDefs, driver });
 ----
 
-== Array comparison
+=== Array comparison
+
+// We should add examples in this section
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/queries-aggregations/filtering.adoc]: ^

File: modules/ROOT/pages/queries-aggregations/filtering.adoc [hunk 2]
```diff
@@ -183,6 +263,7 @@ const neoSchema = new Neo4jGraphQL({ features, typeDefs, driver });
 
 For both `String` and `ID`:
 
+// Is the following correct? would have expected "filters: { String : { MATCHES: true, ID: true}}" based on the two above
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/queries-aggregations/filtering.adoc]: ^

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-180.surge.sh

Updated pages:
- [Filtering](https://neo4j-docs-graphql-180.surge.sh/graphql/5/queries-aggregations/filtering)

File: modules/ROOT/pages/queries-aggregations/filtering.adoc [hunk 3]
```diff
@@ -3,12 +3,50 @@
 :page-aliases: filtering.adoc
 :description: This page describes filtering operators.
 
-When querying for data, a number of operators are available for different types in the `where` argument of a query or mutation.
+When querying for data, a number of operators are available for the types in the `where` argument of a query or mutation, allowing you to filter query results.
```
Comment (mjfwebb) [inline modules/ROOT/pages/queries-aggregations/filtering.adoc]: > allowing you to filter query results.

The filter for a query filters query results, but the filter for a mutation narrows the changes you will make. I think the wording here should be changed.

Comment (mjfwebb) [reply modules/ROOT/pages/queries-aggregations/filtering.adoc]: i.e. "update all Cars that are the colour red" would be the natural language version of a mutation filter

File: modules/ROOT/pages/queries-aggregations/filtering.adoc [hunk 4]
```diff
@@ -3,12 +3,50 @@
...
+        }
+    }
+}
+----
+
+`name_CONTAINS` and `name_ENDS_WITH` are xref:/queries-aggregations/filtering.adoc#_string_comparison[String comparisons] while `movies_SOME` is a xref:/queries-aggregations/filtering.adoc#_relationship_filtering[relationship filter]. 
+
+=== Equality operators
+
+All types can be tested for equality or non-equality.
```
Comment (mjfwebb) [inline modules/ROOT/pages/queries-aggregations/filtering.adoc:49]: Missing a new line afterwards perhaps?

File: modules/ROOT/pages/queries-aggregations/filtering.adoc (hunk 2 above)
Comment (mjfwebb) [reply modules/ROOT/pages/queries-aggregations/filtering.adoc]: No, the above is actually incorrect.

Correct examples:

```ts
features: {
    filters: {
        ID: {
            MATCHES: true,
        },
    },
},
```


```ts
features: {
  filters: {
      String: {
          MATCHES: true,  
      },
      ID: {
          MATCHES: true,
      },
  },
},
```

File: modules/ROOT/pages/queries-aggregations/filtering.adoc (hunk 3 above)
Comment (rsill-neo4j) [reply modules/ROOT/pages/queries-aggregations/filtering.adoc]: added a phrase for mutations

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-180.surge.sh

Updated pages:
- [Filtering](https://neo4j-docs-graphql-180.surge.sh/graphql/5/queries-aggregations/filtering)

**Review by mjfwebb — APPROVED** [review]

**Pull request merged**